File: livingrimoire start here/LivinGrimoire java/src/Auxiliary_Modules/AlgorithmV2.java
package Auxiliary_Modules;

import LivinGrimoire.Algorithm;

public class AlgorithmV2 {
    private int priority;
    private Algorithm alg;

    public AlgorithmV2(int priority, Algorithm alg) {
        this.priority = priority;
        this.alg = alg;
    }

    public int getPriority() {
        return priority;
    }

    public void setPriority(int priority) {
        this.priority = priority;
    }

    public Algorithm getAlg() {
        return alg;
    }

    public void setAlg(Algorithm alg) {
        this.alg = alg;
    }
}

File: livingrimoire start here/LivinGrimoire Kotlin/kotlin/auxiliary_modules/AlgorithmV2.kt
package auxiliary_modules

import livinGrimoire.Algorithm

class AlgorithmV2(priority: Int, alg: Algorithm) {
    var priority = 4
    private var alg: Algorithm? = null

    init {
        this.priority = priority
        this.alg = alg
    }

    fun getAlg(): Algorithm? {
        return alg
    }

    fun setAlg(alg: Algorithm?) {
        this.alg = alg
    }
}